src/_derive/_tutorial.rs

```diff
@@ -152,6 +153,16 @@
 //! ```
 #![doc = include_str!("../../examples/tutorial_derive/03_05_default_values.md")]
 //!
+//! ### Globals
+//!
+//! We've previously showed subcommands, but sometimes you want an argument to interact with any subcommand.
+//! In this case a `#[clap(global)]` argument can be specified, allowing it to be used from any position.
+//!
+//! ```rust
+#![doc = include_str!("../../examples/tutorial_derive/03_06_globals.rs")]
+//! ```
+#![doc = include_str!("../../examples/tutorial_derive/03_06_globals.md")]
+//!
```

Comment:
Please open an issue to discuss this.  We need to decide if and how we want this included.  Conversations like that are best left to issues (with PRs focused on implementation) as sometimes multiple PRs get created/closed to address one problem and it scatters the conversation

Reply:
You want me to contribute… less?

Should I just pull everything out except for the reference-level mention?

Reply:
> You want me to contribute… less?

No but modifying a tutorial is not a trivial addition.  Tutorials have to maintain a specific scope and narrative and changes to that need to be taken with care.  As I said, discussing higher level requirements works best in issues.